eyelid-match-ops/src/primitives/yashe/test/encdec.rs

```diff
@@ -28,4 +29,5 @@ fn encrypt_decrypt_test() {
     // TODO: get TinyTest working here
     encrypt_decrypt_helper::<MiddleRes>();
     encrypt_decrypt_helper::<FullRes>();
+    encrypt_decrypt_helper::<LargeRes>();
```

Comment:
Is this why the tests are taking 20 minutes?

Maybe we could run it in a separate test function, and turn it off by default?
```rust
#[test]
#[ignore]
fn encrypt_decrypt_test_large() ...
```

Reply:
No expensive test is `encrypted::test::matching::test_matching_homomorphic_codes`.